Capstone Project

Yazino12 opened the pull request:
**See latest video link here: [Link](https://www.loom.com/share/60c72699a9ea473694b3a3e35de7002a)**

### Changes made:

- Create two pages for mobile
- Adapt two pages for Desktop
- Add featured section dynamically
- Implement Screen size appearance differences
- Generate features according to screen size and add more button functionality
- Implement navbar sticky and mobile menu functionality
- Add CSS animation to logo, title and feature images
- Modify background for home page and add background to the title

### New Changes made: 

- Add about page and sections
- Add styles and javascript behaviour for the about page menu
- Add nav top section, partner section and footer section

### New Changes made: 

- Remove partner section from about page
- Change footer background colour for about page

Yazino12 committed:
Add tools and setup

Yazino12 committed:
Create two pages for mobile
Add nav, home and about sections

Yazino12 committed:
Adapt two pages for Desktop
Add styles to adapt screens 768px and larger
Add transition animations and hover affects

Yazino12 committed:
Implement featured section with dynamic html
Generate featured section from array data and create dynamic html content
Add and adjust styles to both mobile and desktop screens

Yazino12 committed:
Implement final functionality features
Implement Screen size appearance differences
Generate features according to screen size and add more button functionality
Implement navbar sticky and mobile menu functionality
Add css animation to logo, title and feature images
Modify background for home page and add background to title
Fix linter errors

Yazino12 committed:
Modify issues in image links for the project

Yazino12 committed:
Modify and add latest images for README file

ReshyResh commented on index.html:
```diff
@@ -0,0 +1,136 @@
+      </section>
+    </main>
+    <script src="index.js"></script>
+  </body>
```
- [x] The main section is missing the 'Partners' section and the footer, as shown in the picture below. Please make sure to add it, to better match the given template.

![](https://i.imgur.com/nGEMxtg.png)

Yazino12 replied:
To be honest, I understood pages as sections 🤦‍♂️, but now that you made it clear I'll implement these as well.

ReshyResh commented on index.html:
```diff
@@ -0,0 +1,136 @@
+    </div>
+
+    <!-- NAVIGATION BAR -->
+    <nav>
+      <i class="fa-solid fa-bars fa-2xl burger-menu"></i>
+      <div class="hidden">
+        <a href="#home"><i class="fa-solid fa-laptop-code fa-2xl logo"></i></a>
+        <ul>
+          <li><a class="menu-item1" href="#home">Home</a></li>
+          <li><a class="menu-item2" href="#about">About</a></li>
+          <li><a class="menu-item3" href="#featured">Featured</a></li>
+        </ul>
+      </div>
+    </nav>
```
- [x] The navigation bar is currently missing the top section, as pictured in the template (The one with the social icons), please make sure to add it.

![](https://i.imgur.com/JStFswR.png)

Yazino12 replied:
Thanks, @ReshyResh for suggesting these factors. I will fix these right away 😊

ReshyResh reviewed (changes requested):
# Hi @Yazino12 , 

### Good job so far with your capstone!
  - ✔ Design matches the given template.
  - ✔ Professional looking repository with good readme and commit messages.
  - ✔ The dynamic content is generated correctly.
  - ✔ Proper use of media queries for different screen sizes.
  - ✔ No linter errors.


#### It seems like you forgot some requirements in order to progresso to the next module, but you're almost there!

## Required Changes ♻️

- [x] As per requirement, in this capstone you're supposed to build 2 web pages, the `index` and the `about` page, both present in the Behance template. Your website currently only has the `index` page, please make sure to implement the about page as well, matching it as close as possible to the given template.
- [x] For the same reason, I'm going to ask you to record another video showcasing your project once again after implementing the requested changes, even though you did an amazing job with the previous presentation!

_Please check the comments under the review for more changes._


Cheers and Happy coding! 👏👏👏

----

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Yazino12 committed:
Add about page and sections
Add styles and javascript behavior for about page menu
Add nav top section
Add partner section and footer section

Yazino12 committed:
Merge branch 'master' into developer

Yazino12 committed:
Update README.md

Yazino12 committed:
Update README.md

Yazino12 committed:
Modify home routing from about page

Yazino12 committed:
Modify home routing from about page attempt 2

Yazino12 committed:
Fix some Mixed Content error preventing font to load

Gambit142 commented on about.html:
```diff
@@ -0,0 +1,161 @@
+
+    <!-- NAVIGATION BAR -->
+    <section class="nav-top">
+      <div>
+        <i class="fa-brands fa-facebook-f"></i>
+        <i class="fa-brands fa-twitter"></i>
+        <span>English</span>
+        <span>My page</span>
+        <span>Logout</span>
+      </div>
```
- [x] (Optional) This is not a hard requirement, you can change the `div` tag to a `nav` tag for HTML semantic purposes. It is still a nav bar although it is the upper level. 👍

Gambit142 commented on index.js:
```diff
@@ -0,0 +1,167 @@
+const populate = function (number) {
+  function generate(items) {
+    items.forEach((feature) => {
+      const content = ` <div class="featured-items">
```
- [x] (Optional) It would be nice to change the `div` tag to an `article` tag for semantic purposes 👍

Gambit142 commented on index.js:
```diff
@@ -0,0 +1,167 @@
+    items.forEach((feature) => {
+      const content = ` <div class="featured-items">
+                          <div class="lefts">
+                            <img src=${feature.thumbnail} alt="Tutor-image" />
+                          </div>
+                          <div class="rights">
```
- [x] (Optional) Your design is one of the best I have seen for this capstone. It would be nice to make it better by making the image placeholder for each card look like the one used in the template design provided (the chessboard background). Remember that your project should look very identical to the template provided by Behance 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/157916038-42992823-8903-46c4-8c1b-30a7c0b01ac1.png)

### Behance Design

![image](https://user-images.githubusercontent.com/77106962/157916132-0a0a5367-357b-4e05-a911-399952b2d6bf.png)

Gambit142 commented on about.html:
```diff
@@ -0,0 +1,161 @@
+          <span><i class="fab fa-free-code-camp"></i>freeCodeCamp</span>
+        </div>
+      </section>
+      <footer>
+        <div class="left">
+          <i class="fa-solid fa-laptop-code fa-2xl"></i>
+          <h3>Bootcamp</h3>
+        </div>
+        <div class="right">
+          <span>2022 Web Dev Malmo. All Rights Reserved</span>
+        </div>
+      </footer>
```
- [x] (Required) It was hard making this a required change because you designed this project nicely. The `footer` for the _**about page**_ should have a `background-color` of `#272a31` and not a white background color. It would be nice to fix this change because your project should look almost identical to the original design, outside the content differences.

- [x] Kindly remove the **_partner_** section from the _**about page**_ because the template design doesn't have the _**partner section**_. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/157918072-109b452a-5740-4b23-8e38-603e9e9faeac.png)

### Behance Design

![image](https://user-images.githubusercontent.com/77106962/157918270-a6f572fa-e63c-40f8-8b57-45d5eda56418.png)

Yazino12 replied:
@Gambit142 Thanks for addressing these suggestions, I will make sure to Implement all of them understanding that they will improve my project a lot. Again Thank you for spending the time and stopping by 🙏🏾.

Gambit142 replied:
You welcome any day anytime @Yazino12 😄

Gambit142 reviewed (changes requested):
Hi @Yazino12 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- This is arguably the best project I have seen from your cohort 👏 
- Linters checks pass 💯 
- Good implementation of UI/UX 💯 
- Good communication with the previous reviewer 💯 

 ## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions

- It would be nice to include a `Getting Started` Section in your `README.md` file where you explain to people who visit your repository the basic steps to get this project running on their local machine. You can check out the `README.md` file of this [repository](https://github.com/Fsher07/Html-Css-JavaScript-CapstoneProject)

- _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Yazino12 committed:
Modify about page
Remove partner section from about page
Change footer background color for about page

prolajumokeoni reviewed (approved):
Hi @Yazino12 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![approved-stamp](https://user-images.githubusercontent.com/69638013/158006330-e73fa4fd-28eb-4cb3-8e1b-f0679eab4a0e.gif)



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
